tests/unit/test_launchers.py

```diff
@@ -105,10 +106,19 @@ def test_launcher_deleted_remove_tempdir(self):
 
 
 class TestFennecLauncher(unittest.TestCase):
+
+    test_root = '/sdcard/tmp'
+
+    def setUp(self):
+        self.profile = Profile()
+        self.addCleanup(self.profile.cleanup)
+        self.profile_name = os.path.basename(self.profile.profile)
```

Comment:
Urgh. Sorry. But the test_root needed to be a class member instead of reaffected on each test run with setUp. I fix profile_name aswell.